doc/widgets/documentembedding.md

```diff
@@ -0,0 +1,50 @@
+Document Embedding
+==================
+
+Embeds documents from input corpus into vector space by using pretrained
+[fastText](https://fasttext.cc/docs/en/crawl-vectors.html) models described in 
+E. Grave, P. Bojanowski, P. Gupta, A. Joulin, T. Mikolov,
+Learning Word Vectors for 157 Languages.
+Proceedings of the International Conference on Language Resources and Evaluation, 2018.
+
+**Inputs**
+
+- Corpus: A collection of documents.
+
+**Outputs**
+
+- Corpus: Corpus with new features appended.
+
+**Document Embedding** parses ngrams of each document in corpus, obtains embedding 
+for each ngram using pretrained model for chosen language and obtains one vector for each document by aggregating ngram embeddings using one of offered aggregators. Note that method will work on any ngrams but it will give best results if corpus is preprocessed such that ngrams are words (because model was trained to embed words).
+
+![](images/Document-Embedding-stamped.png)
```

Comment:
Sorry that I didn't tell you before. For putting numbers on the image from the documentation we use stamper.py script from this repository: https://github.com/biolab/orange-tools/tree/master/tools
It will produce a stamped image that is the same as other images in the documentation.

This repository also contains some other useful scripts. One very useful is trim which removes the background from the image. You make a screenshot on a solid desktop background (e.g. red/ green) and then just run the trim script and background is removed.